src/about-us/index.html

```diff
@@ -55,10 +55,10 @@
                     </div>
                     <div class="media_body">
                         <span class="h6">Director</span>
-                        <h3 class="h2 bureau-bio_name">Richard Cordray</h3>
+                        <h2 class="bureau-bio_name">Richard Cordray</h2>
                         <a class="jump-link jump-link__underline"
                            href="/the-bureau/about-rich-cordray">
-                            Cordray's bio
+                            Cordray’s bio
```

Comment:
I think I remember this being a discussion and content usually goes with Cordray vs. Director to help strike a friendlier tone. Lemme tag @sarahsimpson09 to weigh in.